pep/benchmark_test.go

```diff
@@ -719,42 +470,35 @@ func BenchmarkThreeStagePolicySetRaw(b *testing.B) {
 	})
 }
 
-func startPDPServer(p string, b *testing.B) (string, string, *proc, Client) {
+func startServer(p string) (string, string, *os.Process, error) {
 	tmpYAST, err := makeTempFile(p, "policy")
 	if err != nil {
-		b.Fatalf("can't create policy file: %s", err)
+		return "", "", nil, fmt.Errorf("cannot create policy file: %s", err)
 	}
 
 	tmpJCon, err := makeTempFile(benchmarkContent, "content")
 	if err != nil {
 		os.Remove(tmpYAST)
-		b.Fatalf("can't create content file: %s", err)
+		return "", "", nil, fmt.Errorf("cannot create content file: %s", err)
 	}
 
-	pdp, err := newProc("pdpserver", "-p", tmpYAST, "-j", tmpJCon)
+	process, err := startProcess("../build/pdpserver", "-p", tmpYAST, "-j", tmpJCon)
```

Comment:
The relative path implies particular directory to run benchmark. I'd change to $GOPATH/src/github.com/infobloxopen/themis/build/pdpserver